website/project/views/register.py

```diff
@@ -475,13 +471,32 @@ def node_register_template_page_post(auth, node, **kwargs):
     push_status_message('Files are being copied to the newly created registration, and you will receive an email notification containing a link to the registration when the copying is finished.')
 
     return {
-        'status': 'initiated',
-        'urls': {
-            'registrations': node.web_url_for('node_registrations')
-        }
+        'status': 'success',
+        'result': register.url,
     }, http.CREATED
 
 
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def node_draft_template_page_post(auth, node, **kwargs):
+    if settings.DISK_SAVING_MODE:
+        raise HTTPError(
+            http.METHOD_NOT_ALLOWED,
+            redirect_url=node.url
+        )
+
+    # TODO: Using json.dumps because node_to_use.registered_meta's values are
+    # expected to be strings (not dicts). Eventually migrate all these to be
+    # dicts, as this is unnecessary
+    draft = node.register_node(auth)
+
+    return {
+               'status': 'success',
```

Comment:
The closing brace here should match the indentation level of the `return` statement.